test cases/common/33 run program/meson.build

```diff
@@ -1,10 +1,6 @@
 project('run command', version : run_command('get-version.py', check : true).stdout().strip(), meson_version: '>=0.1.0')
 
-if build_machine.system() == 'windows'
-  c = run_command('cmd', '/c', 'echo', 'hello', check: false)
-else
-  c = run_command('echo', 'hello', check: false)
-endif
+c = run_command('echo', 'hello', check: false)
```

Comment:
This will instead break any Windows system that doesn't have a copy of MSYS2 installed on PATH...

Reply:
Done at fd50cee

Reply:
Please don't include commits that `git revert` a previous commit in the same PR -- instead the commit should be removed from the PR so that it is no longer listed in the Commits tab (e.g. via `git rebase -i` and deleting the line for that commit). Then code a replacement using the "one commit per logical change" axiom.

Reply:
reverted commit now removed